### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

**Comment by BramGruneir** [comment]
LGTM

File: proto/internal.proto [hunk 1]
```diff
@@ -147,7 +147,7 @@ message InternalPushTxnRequest {
 // otherwise.
 message InternalPushTxnResponse {
   optional ResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
-  // Txn is non-nil if the transaction could be heartbeat and contains
+  // pushee_txn is non-nil if the transaction could be heartbeat and contains
```
Comment (spencerkimball) [inline proto/internal.proto]: s/could be heartbeat/was pushed/

Comment (tamird) [inline proto/internal.proto]: fixed

File: storage/store.go [hunk 2]
```diff
@@ -1372,7 +1372,14 @@ func (s *Store) resolveWriteIntentError(ctx context.Context, wiErr *proto.WriteI
...
 	// We pushed the transaction(s) successfully, so resolve the intent(s).
 	for i, intent := range wiErr.Intents {
-		pushReply := bReply.Responses[i].GetValue().(*proto.InternalPushTxnResponse)
+		pushReply := bReply.Responses[i].InternalPushTxn
+		// TODO(tamird): remove after #1483 is fixed
+		if pushReply == nil {
+			panic("pushReply is nil")
+		}
+		if pushReply.PusheeTxn == nil {
+			panic("pushReply.PusheeTxn is nil")
```
Comment (tamird) [inline storage/store.go]: fwiw this is the one we hit

**Comment by tschottdorf** [comment]
Those assertions are unlikely to help you much. Try asserting whenever a command is retried (`DistSender`, `Store`), and when it comes out of `r.executeCmd`.

**Comment by tschottdorf** [comment]
And those assertions that you do have, definitely print out `pushReply` with `#+v`. That could help a lot.

**Pull request merged**